File: Codeforces/Golang/800-1000/705A - Hulk/main.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	var n int
	fmt.Scan(&n)

	var sb strings.Builder

	for i := 1; i <= n; i++ {
		if i%2 != 0 {
			sb.WriteString("I hate")
		} else {
			sb.WriteString("I love")
		}

		if i == n {
			sb.WriteString(" it")
		} else {
			sb.WriteString(" that ")
		}
	}
	fmt.Println(sb.String())
}

File: Codeforces/Rust/800-1000/705A - Hulk/src/main.rs
use std::io;

fn main() {
    let mut input = String::new();
    let _ = io::stdin().read_line(&mut input);
    let n: i64 = input.trim().parse().unwrap();

    let mut result = Vec::new();

    for i in 1..=n {
        if i % 2 != 0 {
            result.push("I hate");
        } else {
            result.push("I love")
        }
    }

    println!("{} it", result.join(" that "))
}
